src/sequence/ner_datadeps.jl

```diff
@@ -0,0 +1,29 @@
+function ner_datadep_register()
+    register(DataDep("NER Model Weights",
+        """
+        The weights for NER Sequence Labelling Model.
+        """,
+        "https://raw.githubusercontent.com/Ayushk4/Random_set_of_codes/weights/weights.tar.xz",
```

Comment:
does `.xz` files work on windows?

Reply:
I don't get what you mean by work?

Reply:
If you mean how to extract/unzip the file, I think `WinRar` should let you extract it.
Or if you are referring to the file being unpacked, then `DataDeps` package function `unpack` handles it.

Reply:
I meant will DataDeps be able to unzip the file without the user having to install additional things?

Reply:
It should work - DataDeps.jl has a test for checking the `unpack` for tarballs. In this line - https://github.com/oxinabox/DataDeps.jl/blob/12bf837dc4ba09edca19fe617d1408ec7dae1796/test/examples.jl#L137

Reply:
I added `Windows` to the os in travis CI a sent a PR to my own fork of `DataDeps.jl`. The tests was ran on `linux`, `mac osz` and `windows` and passed on all the os - https://github.com/Ayushk4/DataDeps.jl/pull/1